src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,78 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  );
+  &__box {
+    margin: auto;
+    border: 7px solid rgba(255, 255, 255, 0.75);
+    border-radius: 59px;
+    padding: 10px;
+    display: table;
+    // height: 1400px;
+    width: 80%;
+    @include responsiveness.screen(smallest, phone, desktop, big-desktop) {
+      width: 60%;
+    }
+  }
```

Comment:
We may need to handle the scaling a little more on screen sizes smaller than 350px. Specifically we want to support a screen size larger than the galaxy fold. If we can get support for galaxy fold that would be great.